.docker/adaa/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:experimental
+FROM maven:3.8.6-jdk-11
+
+WORKDIR /app
+ADD . /app
+
+RUN mvn clean install
+RUN chmod 0777 -R /app
```

Comment:
I'd recommend changing this line. However, I couldn't get past the permissions issue. That's why I left it here.